django_x509/base/views.py

```diff
@@ -8,7 +8,7 @@ def crl(request, pk):
     """
     returns CRL of a CA
     """
-    if app_settings.CRL_PROTECTED and not request.user.is_authenticated():
+    if app_settings.CRL_PROTECTED and not request.user.is_authenticated:
```

Comment:
this would produce wrong results on older versions of django.

use the following:

```python
authenticated = request.user.is_authenticated
authenticated = authenticated() if callable(authenticated) else authenticated
if app_settings.CRL_PROTECTED and not authenticated:
```